test/unit/utils.test.js

```diff
@@ -222,4 +222,15 @@ describe('Utils functions', () => {
       }
     );
   });
+
+  describe('makeAgeGroupPlural', () => {
```

Comment:
What happens if you pass in a number to `utils.makeAgeGroupPlural`? Or don't pass anything at all? For the latter, I think you'd get `undefined` as the returned value.